src/pages/LandingPage.jsx

```diff
@@ -1,9 +1,49 @@
 import React from 'react';
+import '../stylesheets/LandingPage.scss';
+import Logo from '../assets/Light_Logo.png';
+import LogoText from '../assets/Light_Logo_Text.png';
+import HYLButton from '../components/HYLButton.jsx';
+
 
 const LandingPage = () => (
-  <div>
-    Landing Page
-  </div>
+  <>
+    {/* Replace this div with actual navbar */}
+    <div className="tempNavBar">
```

Comment:
The Navbar component will be included in the `Routes.jsx` file, so no need to include it here